src/main/java/net/glowstone/entity/GlowLivingEntity.java

```diff
@@ -997,8 +998,22 @@ public TaskManager getTaskManager() {
     public boolean entityInteract(GlowPlayer player, InteractEntityMessage message) {
         super.entityInteract(player, message);
 
+        if (message.getAction() != Action.INTERACT.ordinal()) {
```

Comment:
I actually want to return false, since the super class does the same in every case: https://github.com/GlowstoneMC/Glowstone/blob/master/src/main/java/net/glowstone/entity/GlowEntity.java#L1395